apps/cli/src/logger.ts

```diff
@@ -14,6 +15,92 @@ interface LogChunk {
   ops?: Array<{ value: string }>;
 }
 
+/**
+ * Format a tool call arguments into a clean, readable string
+ */
+function formatToolCallArgs(tool: any): string {
```

Comment:
```suggestion
function formatToolCallArgs(tool: ToolCall): string {
```

and just redefine it (langchain.js doesnt export it) to this:
```typescript
// TODO: Delete whenever we upgrade to LangChain 1.0, and replace with new type
export type ToolCall = {
  name: string;
  args: Record<string, any>;
  id?: string;
  type?: "tool_call";
};
```